content/en/user-manual/graphics/cameras/depth-layer.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Depth Layer
+layout: usermanual-page.hbs
+position: 2
+---
+
+Some rendering techniques require access to the depth or the color buffer of the scene for a specific camera. The Depth Layer is a special layer, which can be added to the `layers` property of a camera, to define at which point during the rendering, the depth or the color buffer is captured. The captured buffers can then be used in the following layers of the camera.
```

Comment:
This makes it sound like adding the depth layer to the camera decides when the depth is captured to a buffer. Isn't it defined by the render layer order? Or is this more about camera stacking?

Reply:
makes sense, adjusted, thanks